modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/WebUrlResourceTest.java

```diff
@@ -113,6 +113,26 @@ protected Long testGetAccountWebUrlsPage_getAccountId() throws Exception {
 		return _accountEntry.getAccountEntryId();
 	}
 
+	@Override
+	protected WebUrl
+			testGetOrganizationByExternalReferenceCodeWebUrlsPage_addWebUrl(
+				String externalReferenceCode, WebUrl webUrl)
+		throws Exception {
+
+		return _addWebUrl(
+			webUrl, _organization.getModelClassName(),
```

Comment:
please use Organization.class.getName() and check if we need to update also the byId method